inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/DataFlowInfo.java

```diff
@@ -75,6 +87,12 @@ public SourceInfo getSourceInfo() {
         return sourceInfo;
     }
 
+    @Nullable
```

Comment:
It's not necessary to add @Nullable

Reply:
@JsonInclude(Include.NON_NULL) is used when serializing the object into json string
@Nullable is used for marking that the field's value can be nulll